src/RealtimeSubscription.ts

```diff
@@ -180,7 +180,14 @@ export default class RealtimeSubscription {
     }
 
     this.bindings
-      .filter((bind) => bind.event === event)
+      .filter((bind) => {
+        // bind all realtime events
+        if (bind.event === '*') {
+          return event === payload.type
```

Comment:
@abc3 I was wondering if it's more explicit and congruous with `bind.event === event` by doing this:

```suggestion
          return ["INSERT", "UPDATE", "DELETE"].indexOf(event) > -1
```

I'm fairly new to `realtime-js` so it might be better to check `payload.type`. Either way, let me know what you think!

Reply:
There is one more event TRUNCATE. I purposely don't check by event name. If will added a new event no need to update client.